tools/cockpit.spec

```diff
@@ -87,11 +87,12 @@ Requires: %{name}-networkmanager = %{version}-%{release}
 Requires: %{name}-ws = %{version}-%{release}
 Requires: %{name}-shell = %{version}-%{release}
 Requires: %{name}-storaged = %{version}-%{release}
+
+# Fedora does not want docker in its default install after F24
+%if 0%{?fedora} == 0 || 0%{?fedora} < 24
 %ifarch x86_64 armv7hl
 Requires: %{name}-docker = %{version}-%{release}
 %endif
-%if 0%{?rhel}
-Requires: %{name}-subscriptions = %{version}-%{release}
```

Comment:
Because it's not necessary. The rhel cockpit-shell includes subscriptions support and doesn't need to depend on it. Do you want me to make this a separate commit?